terraform.tfvars.example

```diff
@@ -42,10 +42,10 @@ control_plane_count = 3
 compute_count = 2
 
 // Ignition config path for the control plane machines
-control_plane_ignition_path = "./master.ign"
+control_plane_ignition_path = "./master.ign" //TODO
```

Comment:
this would be the path to the ignition files you've created with `openshift-install create ignition-configs`.